src/constants.rs

```diff
@@ -1 +1,2 @@
+pub(crate) const NUM_CHALLENGE_BITS: usize = 128;
 pub const N_BITS_CHALLENGE: usize = 250;
```

Comment:
These 2 constants seem to have the same name (with different order), which is the difference between them?